main.tf

```diff
@@ -180,6 +180,25 @@ resource "aws_iam_role_policy" "master_policy" {
 {
     "Version": "2012-10-17",
     "Statement": [
+      {
+          "Sid": "DCOSExternalExhibitorBucketLevel",
+          "Effect": "Allow",
+          "Action": [
+              "s3:ListBucket"
+          ],
+          "Resource": "arn:aws:s3:::${coalesce(var.aws_s3_bucket, "soak-cluster-logs")}"
+      },
+      {
+          "Sid": "DCOSExternalExhibitorObjectLevel",
+          "Effect": "Allow",
+          "Action": [
+              "s3:PutObject",
+              "s3:GetObject",
+              "s3:DeleteObject"
+              
+          ],
+          "Resource": "arn:aws:s3:::${coalesce(var.aws_s3_bucket, "soak-cluster-logs")}/*"
```

Comment:
🤔 "soak-cluster-logs")}/*" ? think this should be deleted.

Reply:
also what happens if there is no bucket defined?

Reply:
So the reason for doing this is just for that reason, "what happens if not bucket is defined?". This will default to "soak-cluster-logs", which is not actually created nor does it exist. 

Without coalesece, if you dont define anything, you end up with 's3:::*' with put, get, and delete. The reason I decided this was because 'soak-cluster-logs' is defined later on in this policy already as a default. Also, created a separate sid for Exhibitor so that I didnt affect the ones already created in case there were plans for these later on. 

I am open to ideas on this of course.

Reply:
wow nice catch... this is a initial copy paste error made by myself.

We should throw out all that stuff... useless for customer clusters.